common/djangoapps/student/models.py

```diff
@@ -2761,3 +2765,22 @@ def update_recovery_email(self, email):
         self.secondary_email = email
         self.is_active = False
         self.save()
+
+    @classmethod
+    def retire_recovery_email(cls, user_id):
+        """
+        Retire user's recovery/secondary email as part of GDPR Phase I
```

Comment:
nit: period or space after this sentence, since sphinx might treat the next line as a continuation.

Reply:
Thanks!